packages/web-app-files/src/components/AppBar/CreateAndUpload.vue

```diff
@@ -274,6 +294,15 @@ export default defineComponent({
       return unref(currentFolder)?.canUpload({ user: store.getters.user })
     })
 
+    const actionKeySuffix = ref(uuidv4())
+    const showDrop = () => {
+      // force actions to be re-rendered when the drop is being opened
+      actionKeySuffix.value = uuidv4()
+    }
```

Comment:
meeeh - good workaround for now, but we should figure out how to make everything properly reactive